ui/src/app/applications/components/application-resource-tree/application-resource-tree.tsx

```diff
@@ -274,11 +274,13 @@ function renderResourceNode(props: ApplicationResourceTreeProps, id: string, nod
                         {node.createdAt || props.app.metadata.creationTimestamp}
                     </Moment>
                 ) : null}
-                {(node.info || []).filter(tag => !tag.name.includes('Resource.')).map((tag, i) => (
-                    <span className='application-resource-tree__node-label' title={`${tag.name}:${tag.value}`} key={i}>
-                        {tag.value}
-                    </span>
-                ))}
+                {(node.info || [])
```

Comment:
Pods now have a lot of "info items": 

![image](https://user-images.githubusercontent.com/426437/102125126-08967e00-3dfe-11eb-95af-5cc073a05b7f.png)

I suggest to show top 4 and add tooltip with all info items.